CMakeLists.txt

```diff
@@ -152,7 +152,22 @@ endif()
 
 if(WIN32)
   message("-- Win32 build detected, setting default features")
-  set(USE_CAMERA_SUPPORT OFF)
+  set(USE_COLORD OFF)
+  set(USE_FLICKR OFF)
```

Comment:
@houz Managed to compile the flickcurl library with a proper shared library, and it links now properly with darktable . I have pushed that to the [MING-packages repository](https://github.com/Alexpux/MINGW-packages/pull/1875). As soon as they merge it there, I'll commit the darktable changes as well.